DEVELOPMENT.md

```diff
@@ -88,6 +96,32 @@ registry which if you are developing locally you could use `export KO_DOCKER_REP
 to use the local one on kind. Please see official
 [KO documentation](https://github.com/google/ko#local-publishing-options) for more information.
 
+#### Running end-to-end tests locally
+
+**NOTE** The `dispatch/path`, `host-rewrite`, and `upgrade` tests seem to fail (as of Jan 28) when run locally on Kind. You'll want to add these to the `--skip-tests` flag in `hack/e2e-tests.sh`.
+
+Run the tests from the root directory; the script will create a _new_ Kind cluster and build and install `net-contour` on it, along with building and pushing test images.
+
+```bash
+./test/e2e-tests.sh --cloud-provider kind
+```
+
+**WHILE THE TESTS ARE RUNNING**, install MetalLB in the cluster:
```

Comment:
This seems a bit janky and I'm curious why we wouldn't just suggest to spin up a kind/metallb cluster prior to running the tests. 

These scripts support skipping the cluster creation as follows
```
./test/e2e-tests.sh --run-tests
```

Reply:
I think the double-invocation nature of the script made me unsure how to get a Kind configuration that matched the script setup. I could say "just install Kind and then MetalLB if that works".